express.js

```diff
@@ -0,0 +1,10 @@
+const express=require("express");
+const app=express();
+
+app.get("/ping",(req,res)=>{
+    res.send("pong")
+})
+
+app.listen(4000,()=>{
```

Comment:
It's a good practice to use environment variables for port numbers. This provides flexibility when deploying to different environments. [important]